workflow.py

```diff
@@ -0,0 +1,27 @@
+import utils
+from search.search_engine import SearchEngine
+import constants
+
+def query_workflow(search, query):
+    short_gresponse = utils.from_gcloud_remove_extra(utils.gcloud_syntax_extraction(query))
+
+    keywords = []
+
+    for token in short_gresponse['tokens']:
+        keywords.append(token['lemma'])
+
+    lan = short_gresponse['lan']
+    synonyms = []
+    if lan == 'es':
+        for word in keywords:
+            for synonym in utils.create_synonym_list_esp(word):
+                synonyms.append(synonym)
```

Comment:
Python has a built-in function for appending lists to lists called `extend`, so you don't have to do this extra for loop for synonyms: https://www.geeksforgeeks.org/append-extend-python/